cleaners/mc.xml

```diff
@@ -33,15 +33,15 @@
 -->
 <cleaner id="midnightcommander">
   <label>Midnight Commander</label>
-  <description>File Manager</description>
+  <description>File manager</description>
   <var name="profile">
     <value os="windows">%UserProfile%\Midnight Commander</value>
     <value os="linux">~/.mc</value>
   </var>
   <option id="history">
     <label>History</label>
     <description>Delete the usage history</description>
-    <action command="delete" search="file" path="$$profile$$/filepos"/>
     <action command="delete" search="file" path="$$profile$$/history"/>
+    <action command="delete" search="file" path="$$profile$$/filepos"/>
```

Comment:
What is the importance of the order?

Reply:
None! I just reordered because I did the patch anyway... "history" on top because the option is "history"... No deeper sense of it!

Reply:
Btw.: Reason for the PR was the "Fix @cleanerby history"...